schema.graphql

```diff
@@ -1,3 +1,111 @@
+# ADD TOTAL TOKENS
+# spread fees are expresed in sell token
+# write a how to query on a specific block
+
+
+######################################################################################################################
+type Totals @entity {
+  "This is a singleton entity to contain accumulators for all values. Id will be always 1"
+  id: ID!
+  "Total number of tokens traded"
+  tokens: BigInt!
+  "Total number of orders placed"
+  orders: BigInt!
+  "Total number of traders"
+  traders: BigInt!
+  "Total number of batches settled"
+  settlements: BigInt!
+  "Total traded volume in USD"
+  volumeUsd: BigDecimal
+  "Total traded volume in ETH"
+  volumeEth: BigDecimal
+  "Total fees in USD"
+  feesUsd: BigDecimal
+  "Total fees in Eth"
+  feesEth: BigDecimal
+}
+
+######################################################################################################################
+type DailyTotals @entity {
+  "Day timestamp"
+  id: ID!
+  "Start day timestamp"
+  timestamp: BigInt!
+  "Total number of tokens traded"
+  tokens: BigInt!
+  "Total number of orders placed"
+  orders: BigInt!
+  "Total number of batches settled"
+  settlements: BigInt!
+  "Total traded volume in USD"
+  volumeUsd: BigDecimal
+  "Total traded volume in ETH"
+  volumeEth: BigDecimal
+  "Total fees in USD"
+  feesUsd: BigDecimal
+  "Total fees in Eth"
+  feesEth: BigDecimal
+  "Traded tokens"
+  tokens: [Token!]!
```

Comment:
shit, we will have an issue now with this one, because we have already one called `tokens` that is a bigint

Maybe we can add total to the counters?: `totalTokens`, `totalOrders`, `totalSettlements`